student.rb

```diff
@@ -1,16 +1,23 @@
 require './person'
+require './classroom'
 
 class Student < Person
-  def initialize(age, classrooom, name = 'Unknown', parent_permission: true)
+  attr_reader :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission:)
-    @classrooom = classrooom
+    @classroom = classroom
+    classroom.add_student(self)
   end
```

Comment:
- [ ] When setting the classroom for a student it also adds it to the classrooms' students, kindly let's add a method that serves as the 'belongs-to' side (a student belongs to a classroom). You may check this [article](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships`). Your method may look like this ⬇️ 

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```